rest-api/model/biobank_order.py

```diff
@@ -50,6 +54,12 @@ class BiobankOrder(Base):
   # in the resulting JSON.
   finalizedUsername = Column('finalized_username', String(255))
 
+  # cancelled finalized order may still be shipped to biobank for destruction
+  orderStatus = Column('order_status', Enum(BiobankOrderStatus))
+  cancelledTime = Column('cancelled_time', UTCDateTime)
+  # a cancelled or edited order must have a reason
```

Comment:
You should indicate whether this gets set on the new or the old order.

Reply:
done